vtex-sites/storecomponents.store
# Fix | First selected filter removed from filters list (and updated whole package.json)

## What's the purpose of this pull request?
When the user remove the last filter (root department) from the search page, nothing happens.

## How it works? 
Now the last filter will not appear to the user, because these changes search the "root department"(last filter) and remove it from facets list.

## How to test it?
In a store, select a department at the navbar. In the search page, see if the department selected appears in the filter list.
